src/main/java/org/reaktivity/reaktor/internal/LabelManager.java

```diff
@@ -72,7 +72,7 @@ public String lookupLabel(
         return labels.get(labelId - 1);
```

Comment:
Thought of synchronizing only checkSnapshot(), but then labels.get() would give wrong result. For e.g one thread does labels.clear() and other is calling labels.get()